src/__snapshots__/shared-render.test.js.snap

```diff
@@ -1,23 +1,47 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`deep: output matches snapshot 1`] = `
-VNode {
-  "attributes": undefined,
-  "children": Array [
-    "second first",
-  ],
-  "key": undefined,
-  "nodeName": "div",
-}
+exports[`deep: context matches snapshot 1`] = `
+preact-render-spy (1 nodes)
+-------
+<div>second first</div>
+
+`;
+
+exports[`deep: find matches snapshot 1`] = `
```

Comment:
if we had snapshotted `.find('div')` or even the `context` instead of `.find('span')` yeah :)